app.py

```diff
@@ -89,22 +138,36 @@ def get_images_to_scan_in_page(page_id, headers):
 
     for index, block in enumerate(results):
         if block['type'] == 'image':
+            if block['image']['caption']:
+                for indexCaption, caption in enumerate(block['image']['caption']):
+                    if caption['plain_text'] and 'ocr_text' in caption['plain_text'].split('\n'):
+                        caption_index = indexCaption
```

Comment:
Suggesting to make this more 'pythonic' by changing the camelCase into a snake_case (eg index_caption)